packages/at_commons/lib/src/keystore/at_key.dart

```diff
@@ -38,22 +38,35 @@ class AtKey {
   /// These keys will never be synced between the client and secondary server.
   bool _isLocal = false;
 
+  String? get key => _key;
+
+  set key(String? key) {
+    _key = key?.toLowerCase();
```

Comment:
set to lower case only if passed key is not null

Reply:
addressed as part of latest commit